src/ansible_navigator/configuration_subsystem/definitions.py

```diff
@@ -154,6 +200,11 @@ class ApplicationConfiguration:
 
     initial: Any = None
 
+    @property
+    def application_name_dashed(self) -> str:
+        """Generate a dashed version of the application name"""
```

Comment:
(PEP 257)
```suggestion
        """Generate a dashed version of the application name."""
```

Reply:
Fixed in incoming commit.

Reply:
BTW.  PEP 257 and any linters that check for it are not currently enforced, PR comments like this should be noted as optional, informational, educational and may bring attention to the ongoing clean up work. We can do better than simply referring to a PEP.

Reply:
Well, it's not like I'm asking to do any work. I've provided a suggestion that can be accepted with a single click in the UI. And it's a one char change so I don't see why it's hard to just accept the suggestion and forget about it.